swift/ql/src/qlpack.yml

```
@@ -0,0 +1,10 @@
+name: codeql/swift-queries
+version: 0.0.0
+groups:
+  - swift
+  - queries
+suites: codeql-suites
```

I have this line as well:
```suggestion
suites: codeql-suites
extractor: swift
```